commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitControllers.scala

```diff
@@ -1,12 +1,12 @@
 package com.lynbrookrobotics.potassium.commons.drivetrain
 
+import java.io.{File, PrintWriter}
```

Comment:
Uhhh this concerns me are we writing to files in our control loops?

Reply:
Oh, oops. I was logging stuff, then I removed the logging.